src/plugins/uploader.js

```diff
@@ -119,31 +120,32 @@ class Uploader {
 		return configOption * 1024;
 	}
 
+	// Returns false if an error occurred (file not found)
```

Comment:
`string | false | null`, that's kind of a tricky return. What about returning an `Error` when an error occurred? Could also have a custom error instead of `null`, but that's fine.

Reply:
I changed it to `?string`.

Returns null on error, and the string can be empty if no type was detected.